src/main/java/org/apache/commons/text/similarity/FuzzyScore.java

```diff
@@ -47,7 +72,8 @@
      */
     @Override
     public Integer compare(CharSequence term, CharSequence query) {
-        return compare(term, query, Locale.getDefault());
+        Locale locale = this.locale != null ? this.locale : Locale.getDefault();
```

Comment:
I'm not sure.   What if Locale.setDefault(Locale) is called afterward?   Shouldn't it switch to using the new Locale?

Please confirm.  I would also remove the DEFAULT_INSTANCE in that case.

Thanks!